layers/+spacemacs/spacemacs-defaults/keybindings.el

```diff
@@ -14,80 +14,81 @@
-                                       ("jl"  "avy line")
-                                       ("js"  "split sexp")
-                                       ("jw"  "avy word")
-                                       ("k"   "lisp")
-                                       ("kd"  "delete")
-                                       ("kD"  "delete-backward")
-                                       ("k`"  "hybrid")
-                                       ("m"   "major mode commands")
-                                       ("n"   "narrow/numbers")
-                                       ("N"   "navigation")
-                                       ("o"   "user bindings")
-                                       ("p"   "projects")
-                                       ("q"   "quit")
-                                       ("r"   "registers/rings/resume")
-                                       ("s"   "search/symbol")
-                                       ("sa"  "ag")
-                                       ("sg"  "grep")
-                                       ("sk"  "ack")
-                                       ("sp"  "search project")
-                                       ("sP"  "search project w/input")
-                                       ("sr"  "ripgrep")
-                                       ("st"  "pt")
-                                       ("sw"  "web")
-                                       ("t"   "toggles")
-                                       ("tC"  "colors")
-                                       ("tE"  "editing-styles")
-                                       ("tEe" "emacs (holy-mode)")
-                                       ("tEh" "hybrid (hybrid-mode)")
-                                       ("th"  "highlight")
-                                       ("tm"  "modeline")
-                                       ("tt"  "timeclock")
-                                       ("T"   "UI toggles/themes")
-                                       ("C-t" "other toggles")
-                                       ("u"   "universal arg")
-                                       ("v"   "expand region")
-                                       ("w"   "windows")
-                                       ("wc"  "centered")
-                                       ("wp"  "popup")
-                                       ("x"   "text")
-                                       ("xa"  "align")
-                                       ("xd"  "delete")
-                                       ("xg"  "google-translate")
-                                       ("xj"  "justification")
-                                       ("xl"  "lines")
-                                       ("xR"  "Randomize")
-                                       ("xt"  "transpose")
-                                       ("xw"  "words")
-                                       ("z"   "zoom")))
+(setq spacemacs/key-binding-prefixes (list
```

Comment:
This form is not very nice to maintain, please replace with elisp backquote notation i.e.
```elisp
`((,dotspacemacs-emacs-command-key "M-x")
  ("!"   "shell cmd"))
```